gui/qt/main_window.py

```diff
@@ -912,19 +912,24 @@ def create_receive_tab(self):
         self.receive_address_e.addCopyButton(self.app)
         self.receive_address_e.setReadOnly(True)
         msg = _('Bitcoin Cash address where the payment should be received. Note that each payment request uses a different Bitcoin Cash address.')
-        self.receive_address_label = HelpLabel(_('Receiving address'), msg)
+        receive_address_label = HelpLabel(_('&Receiving address'), msg)
+        receive_address_label.setBuddy(self.receive_address_e)
         self.receive_address_e.textChanged.connect(self.update_receive_qr)
         self.cashaddr_toggled_signal.connect(self.update_receive_address_widget)
-        grid.addWidget(self.receive_address_label, 0, 0)
+        grid.addWidget(receive_address_label, 0, 0)
         grid.addWidget(self.receive_address_e, 0, 1, 1, -1)
 
         self.receive_message_e = QLineEdit()
-        grid.addWidget(QLabel(_('Description')), 1, 0)
+        receive_message_label = QLabel(_('&Description'))
```

Comment:
I presume this is for tabbing.   I actually forget how this works:

So is it the use of the &D accelerator thing that allows tabbing or the buddy or both?

Just wondering...

Reply:
Damn, this is where this code went, I thought I lost it. I accidentally included that. But yes, it is for tabbing.